scripts/predictor/README.md

```diff
@@ -0,0 +1,59 @@
+# WHAT IS THIS SCRIPT SET FOR?
+
+This set of scripts is intended to help devs to find out if the contracts of a given commit / branch in a repo will reproduce exactly the contracts deployed in the blockchain.
+These scripts are categorized into four .js files for convenient separate execution.
```

Comment:
pls add a high level schematic description of how it works as a sequence of steps like
1) provide/generate ABIs and bytecodes of the contracts you want to verify 
2) provide contracts addresses
3) run comparator script on a blockchain (provide as parameter) to get the list of contracts which ABIs correspond to the deployed contracts meaning that they will be successfully verified in the blockchain explorers

Reply:
btw can it be extended with automatic verifier? :)